content/_data/sidebar.json

```diff
@@ -31,27 +27,69 @@
               "slug": "installing-cypress"
             },
             {
-              "title": "End-to-End Testing"
+              "title": "Opening the App",
+              "slug": "opening-the-app"
+            }
+          ]
+        },
+        {
+          "title": "End-to-End Testing",
+          "slug": "end-to-end-testing",
+          "children": [
+            {
+              "title": "Introduction to End-to-End Testing"
             },
             {
-              "title": "Writing Your First Test",
+              "title": "Writing Your First End-to-End Test",
               "slug": "writing-your-first-end-to-end-test"
             },
             {
               "title": "Testing Your App",
               "slug": "testing-your-app"
             },
             {
-              "title": "Component Testing"
+              "title": "Testing Strategies"
+            },
+            {
+              "title": "Auth0 Authentication",
+              "slug": "auth0-authentication"
+            },
+            {
+              "title": "Amazon Cognito Authentication",
+              "slug": "amazon-cognito-authentication"
+            },
+            {
+              "title": "Okta Authentication",
+              "slug": "okta-authentication"
+            },
+            {
+              "title": "Google Authentication",
+              "slug": "google-authentication"
+            },
+            {
+              "title": "Working with GraphQL",
+              "slug": "working-with-graphql"
+            }
+          ]
+        },
+        {
+          "title": "Component Testing",
+          "slug": "component-testing",
+          "children": [
+            {
+              "title": "Introduction to Component Testing"
             },
             {
-              "title": "Overview",
-              "slug": "component-testing"
+              "title": "Testing Your Components with Cypress",
```

Comment:
Is there a way to get this url to be /guides/component-testing? If not, lets make this /guides/component-testing/overview as this is the main landing page in the docs for CT. Afterwards, message Matt Volmer and let him know the URL to here is changing as he'll need to update it in some places.  Thanks

Reply:
I've added the redirects but they don't work on local or preview. I'm hoping that the netlify redirects will kick in on prod so I am resolving this & will revisit after GA